optima/parameters.py

```diff
@@ -126,14 +126,43 @@ def fixprops(self, fix=None, which=None, startyear=None):
         if   which is None:  which = ['tx','supp']
         elif which is 'all': which = ['dx','tx','supp']
         else:                which = promotetolist(which)
-        if fix:  startyear = self.pars['numtx'].t['tot'][-1]
-        else:    startyear = 2100
+        if startyear is None:
```

Comment:
wait, isn't this a different `startyear`? i think this is a bug and should be a different variable name entirely

Reply:
I don't think it's a different `startyear`? It looks like I should be able to call thie method with a startyear, and in the previous implementation, it wouldn't be used...